nixos/modules/services/web-servers/hydron.nix

```diff
@@ -47,16 +77,40 @@ in with lib; {
   };
 
   config = mkIf cfg.enable {
+    security.sudo.enable = cfg.enable == true;
+    services.postgresql.enable = cfg.enable == true;
+
+    services.hydron.passwordFile = mkDefault (toString (pkgs.writeTextFile {
```

Comment:
You can use the shorter `pkgs.writeText "hydron-password-file" cfg.password`. Also `toString` shouldn't be needed.